srv/pom.xml

```diff
@@ -111,7 +111,7 @@
 						</goals>
 						<configuration>
 							<commands>
-								<command>build/all --clean</command>
+								<command>build --for java-cf --clean</command>
```

Comment:
--clean option is not required anymore (should be the default). We should also remove that with this change.